src/functions/memo-management/drive-memos/create-memo-pdf/handler.ts

```diff
@@ -0,0 +1,36 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
```

Comment:
Why do we have method for uploading PDFs into the drive? I would expect this PR to contain only method for posting new suggestion to management team (create new Trello card), not for creating any new memos

Reply:
I planned to use this method as an admin feature to automate the process of pdf files creation from google docs. Otherwise, someone from management team have to download files from google drive and then upload them.